targets/TARGET_NORDIC/TARGET_NRF5x/gpio_api.c

```diff
@@ -129,8 +129,18 @@ static void gpio_apply_config(uint8_t pin)
             cfg.hi_accuracy = false;
             cfg.is_watcher = false;
             cfg.sense = NRF_GPIOTE_POLARITY_TOGGLE;
+            switch(m_gpio_cfg[pin].pull) {
+                case PullUp:
+                    cfg.pull = NRF_GPIO_PIN_PULLUP;
+                break;
+                case PullDown:
+                    cfg.pull = NRF_GPIO_PIN_PULLDOWN;
+                break;
+                default:
+                    cfg.pull = NRF_GPIO_PIN_NOPULL;
+                break;
```

Comment:
@0xc0170 I'm not up to speed on our style guide. Is this correctly formatted?

Reply:
Should be fixed